packages/mds-geography-service/repository/index.ts

```diff
@@ -58,17 +58,38 @@ class GeographyReadWriteRepository extends ReadWriteRepository {
 
   protected findGeographies = async (
     where: FindManyOptions<GeographyEntity>['where'],
-    { includeMetadata = false, includeGeographyJSON = true }: GetGeographiesOptions = {}
+    { includeMetadata = false, includeGeographyJSON = true, includeHidden = false }: GetGeographiesOptions = {}
   ): Promise<GeographyWithMetadataDomainModel[]> => {
     try {
       const connection = await this.connect('ro')
 
       const select = [
-        ...(<const>['geography_id', 'name', 'description', 'effective_date', 'publish_date', 'prev_geographies']),
-        ...(includeGeographyJSON ? <const>['geography_json'] : [])
+        ...(<const>[
+          'g.geography_id',
```

Comment:
what's all this `g` for?

Reply:
I think it's basically a variable name, assigned to one of the two tables that Terence is doing a join on.

Reply:
Yes. This was to disambiguate the shared columns when performing a left join when includeHidden is used.